ui/core/src/hooks/addLiquidityCalculator.ts

```diff
@@ -112,31 +112,52 @@ export function usePoolCalculator(input: {
   const aPerBRatioMessage = computed(() => {
     const aAmount = fromField.fieldAmount.value;
     const bAmount = toField.fieldAmount.value;
-    if (!aAmount || !bAmount) return "";
-    if (bAmount.equalTo("0")) return "";
+    
+    if (!aAmount || aAmount.equalTo("0")) return ""; // invalid, must supply external
+    if (!bAmount || bAmount.equalTo("0")) {
+      // if rowan is 0 or empty ...
+      // allow if the pool exists (BUT ratio is inapplicable - N/A),
+      // invalid if the pool doesn't exist - ""
+      return preExistingPool.value ? "N/A" : ""; 
+    }
+
     return aAmount.divide(bAmount).toFixed(8);
   });
 
   const bPerARatioMessage = computed(() => {
     const aAmount = fromField.fieldAmount.value;
     const bAmount = toField.fieldAmount.value;
-    if (!aAmount || !bAmount) return "";
-    if (aAmount.equalTo("0")) return "";
+
+    if (!aAmount || aAmount.equalTo("0")) return ""; // invalid, must supply external
+
+    if (!bAmount || bAmount.equalTo("0")) {
+      // if rowan is 0 or empty ...
+      // allow if the pool exists (BUT ratio is inapplicable - N/A),
+      // invalid if the pool doesn't exist - ""
+      return preExistingPool.value ? "N/A" : ""; 
+    }
 
     return bAmount.divide(aAmount).toFixed(8);
   });
 
   const state = computed(() => {
+    const aAmount = fromField.fieldAmount.value;
+    const bAmount = toField.fieldAmount.value;
+
     if (!input.fromSymbol.value || !input.toSymbol.value)
       return PoolState.SELECT_TOKENS;
-    if (
-      fromField.fieldAmount.value?.equalTo("0") ||
-      toField.fieldAmount.value?.equalTo("0")
-    )
-      return PoolState.ZERO_AMOUNTS;
-    if (fromBalanceOverdrawn.value || toBalanceOverdrawn.value) {
+    
+    if (fromBalanceOverdrawn.value || toBalanceOverdrawn.value) 
       return PoolState.INSUFFICIENT_FUNDS;
-    }
+
+    if (!aAmount || aAmount.equalTo("0"))
+      return PoolState.ZERO_AMOUNTS;
+
+    if (!bAmount || bAmount.equalTo("0"))
+      // if rowan is 0 or empty ...
+      // allow if the pool exists
+      // invalid if the pool doesn't exist - ""
+      return preExistingPool.value ? PoolState.VALID_INPUT : PoolState.ZERO_AMOUNTS;
 
     return PoolState.VALID_INPUT;
```

Comment:
nit: this might read a little better if every if statement was the expression that resulted in a single state so there should only be one expression that leads to the state ZERO_AMOUNTS